service/policy/objects.proto

```diff
@@ -247,11 +286,17 @@ message ResourceMapping {
 message KeyAccessServer {
   string id = 1;
   // Address of a KAS instance
-  string uri = 2 [(buf.validate.field).cel = {
-      id: "uri_format",
-      message: "URI must be a valid URL (e.g., 'https://demo.com/') followed by additional segments. Each segment must start and end with an alphanumeric character, can contain hyphens, alphanumeric characters, and slashes.",
-      expression: "this.matches('^https?://[a-zA-Z0-9]([a-zA-Z0-9\\\\-]{0,61}[a-zA-Z0-9])?(\\\\.[a-zA-Z0-9]([a-zA-Z0-9\\\\-]{0,61}[a-zA-Z0-9])?)*(:[0-9]+)?(/.*)?$')"
-  }];
+  string uri = 2 [ (buf.validate.field).cel = {
+    id : "uri_format",
+    message : "URI must be a valid URL (e.g., 'https://demo.com/') followed by "
+              "additional segments. Each segment must start and end with an "
+              "alphanumeric character, can contain hyphens, alphanumeric "
+              "characters, and slashes.",
+    expression : "this.matches('^https?://"
```

Comment:
Was splitting up the regex like this a requirement from a new linter? I am not sure the formatting improvement is worth the readability impact to the regex. 🤔

Reply:
I must have a linter locally that did this.